libs/gl-client/src/signer/mod.rs

```diff
@@ -592,6 +605,29 @@ impl Signer {
     pub fn version(&self) -> &'static str {
         VERSION
     }
+
+    /// Create a base64 encoded rune from the master rune of this signer.
+    pub fn create_rune(
+        &self,
+        unique_id: Option<&str>,
+        restrictions: Option<&str>,
```

Comment:
I wonder if passing in the correct types rather than their stringified version wouldn't be better. This way we're likely going to create restrictions, stringifying them, then parsing them again. Also by passing in the correct types we make this pretty much infallible.

Reply:
Oh I forgot that I removed the `create_runes` function from the clients ^^. Yes I can switch back using correct types. Still we might want to think about the enums as signer-less devices will need to ask for restrictions during the pairing and this is something we will likely initiate from the bindings-layer.